src/components/map/types.ts

```diff
@@ -38,6 +38,8 @@ export type MapRegion = {
   center: Position;
 };
 
+export type MappPadding = number | number[];
```

Comment:
Why do you need this type? It seems a bit "dangerous" as the original type (MapboxGL.Padding) enforces the correct number of numbers in the array. This doesn't, and that could cause run-time errors that otherwise would have been discovered at compile time.

Reply:
Hi, that is how the parameter now is expected from the library before it was Padding but is removed:

![image](https://github.com/AtB-AS/mittatb-app/assets/43450882/76202b5a-e4b8-43ad-aaee-9eca21a9d949)

I just created a MapPadding to do not repeat on all the places we use it. 

I will fix the typo ;) thanks

Reply:
Ok. Maybe `MapPadding` should be more like the old `MapboxGL.Padding` then? I.e. `type MapPadding = number | [number, number] | [number, number, number, number]`? I assume that the padding must still consist of a 2 or 4 array?

Reply:
I made as you suggested ;)